docs/user-guide/query/gaffer-syntax/filtering.md

```diff
@@ -0,0 +1,520 @@
+# Filtering Data
+
+One of the key parts of querying data is being able to filter bits out you may
+not want. This page will cover some of the common ways you can filter the
+returned data from an Operation to get the data you want.
+
+In Gaffer the main way you filter data is by applying whats known as a `View` to
+a returned set of elements. A `View` as the name suggests, allows you to view
+the data in a different way this can be either via a filter, aggregation,
+transformation or just general manipulation of the results.
+
+Use cases with a `View` usually fall into one of the following catagories:
+
+- **Filtering** - General filtering on elements based on predicates. Filtering
+    can be applied pre-aggregation, post aggregation and post transformation.
+
+- **Aggregation** - This is to control how similar elements are aggregated
+    together. You can provide a subset of the schema `groupBy` properties and
+    override existing aggregation functions.
+
+- **Transformation** - Transformations can be applied by providing Functions to
+    transform properties and vertex values. This is a powerful feature, you can
+    override the existing values or you can transform and save the new value
+    into a new transient property.
+
+- **Property Removal** - The relevant properties you want to be returned can be
+    controlled. You can use either `properties` or `excludeProperties` to define
+    the list of properties to be included or excluded.
+
+## Filtering in Practice
+
+We will demonstrate general filtering on a query. Take the following graph as a
+basic example, where we have a `Person` node with ID `John` that has a few
+`Created` edges with a `weight` property on them.
+
+```mermaid
+graph LR
+    A(["Person
+
+        ID: John"])
+    --
+    "Created
+     weight: 0.2"
+    -->
+    B(["Software
+
+        ID: 1"])
+    A
+    --
+    "Created
+     weight: 0.6"
+    -->
+    C(["Software
+
+        ID: 2"])
+```
+
+Now say the `weight` property could represent how much a `Person` contributed to
+creating something, and so we wanted to find only significant contributions. To
+do this we can apply a filter to act as a threshold to only get edges with a
+`weight` more than a specific value.
```

Comment:
The whole paragraph doesn't read too well. I would rephrase it to flow a little better. 
```suggestion

Lets say the `weight` property represents how much a `Person` contributed to
creating something and that we wanted to only find the most significant contributions. To
do this we can apply a filter to act as a threshold to only get edges with a
`weight` more than a specific value.
```